simpledotorg/simple-android
# Add facility protocol and group UUID in demo facility

- Protocol UUID is required to open the edit medicines screen when using the demo account
- Group UUID is required to show demo facility in facility picker screen

**Testing steps**
- Log in into the app using demo account details
- Check P0 flows of the app
